### ISSUE TITLE
Gnu head icon link is broken

### ISSUE THREAD

**Issue by charignon** [railwaycat/homebrew-emacsmacport#208]
Thank you for all your work!

It looks like https://github.com/d12frosted/homebrew-emacs-plus/pull/246 broke the gnu head icon reference we have here. Therefore, the default installation instructions don't work and lead to:

```
curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "emacs-mac--gnu-head-icon"
Download failed: https://raw.githubusercontent.com/d12frosted/homebrew-emacs-plus/master/icons/heckert_gnu.icns
Please create pull requests instead of asking for help on Homebrew's GitHub,
```

I am not sure how to fix it, but could send a patch if you don't get to it in the next couple of weeks.

If anyone is hitting that issue, the workaround is to use another icon (the other links seem to work).

**Comment by qhuyduong** [comment]
I'm fixing it in https://github.com/railwaycat/homebrew-emacsmacport/pull/210

**Comment by ibynx** [comment]
@charignon 
> the workaround is to use another icon (the other links seem to work)

Could you expand on the workaround? I'm trying `brew reinstall emacs-mac --with-spacemacs-icon --with-modules --with-natural-title-bar` and it's still trying to download every single icon and failing at the gnu head one. I've also tried the other flags to no avail.

**Comment by d1egoaz** [comment]
@ibynx try this

as emacs is broken/removed we need another editor:
```sh
export EDITOR=vim
```
then edit the local formula:
```
brew edit emacs-mac
```

and finally apply this change manually or using the diff file:
https://github.com/railwaycat/homebrew-emacsmacport/pull/210/files

**Comment by ibynx** [comment]
Thanks!

**Comment by railwaycat** [comment]
I have merged #210, thanks @qhuyduong!
